### ISSUE TITLE
Seems to be broken with PHPStan 0.10.3

### ISSUE THREAD

**Issue by ghost** [BradCrumb/atom-linter-phpstan#3]
Hello

It seems to no longer show any output in Atom. I took the command line that is being run and ran it manually, and this gives the following output:

```
Note: Using the option --errorFormat is deprecated. Use --error-format instead.
```

If I change that, it seems to work again, though I'm not sure the same will work for older PHPStan versions.

**Comment by BradCrumb** [comment]
Hi @tominventisbe ,

Thank you for using my linter.

I reproduced the problem and going to fix it in a couple of days.

**Comment by mallardduck** [comment]
Found the same issue and submitted #5 which seems to fix the error.
However since I just found this package I'm not sure if it actually fixes all the issues.

**Comment by mallardduck** [comment]
Hey @tominventisbe,

A new incarnation of this can be found here: https://atom.io/packages/linter-phpstan
The new repo is at: https://github.com/mallardduck/linter-phpstan

**Comment by BradCrumb** [comment]
@mallardduck Thank you for your interest in my package. May I ask why your created your own fork and package for release management instead of contributing to the original package?

Maybe it is interesting if we combine our interest in this package and improve it together? My version has over 600 downloads at the moment. I'am happy to share the maintanance with you if you want.

**Comment by mallardduck** [comment]
To be blunt, the fork existed so that I could create the PR I submitted ~6 months ago. The fact that the extension was broken and without maintenance for that long meant that either I stop using it or fix it myself. I am just a single affected user in the community - so I shared my efforts to benefit others.

I don't care about maintaining this package, or my fork, or download counts. I care about being a productive PHP and Web developer - so when tools I utilize to accomplish this don't work, it's not a good situation. This was the only motivation for me to release "my own" version.

I would be happy to help maintain this repository and take on managing some Issue/PR related work. I do really enjoy the collaborative nature of FOSS, so it's right up my alley. However I don't know if this fixes the issue fully. I'm not experienced with APM enough to know fore sure how shared packages work. If I would be able to publish to APM as well, then we're all good and I'm just worrying over nothing.

EDIT: I am still sleepy from waking up and realized some of this came off as mean/rude while I only intended it to be factual albeit a bit blunt.
